Project2015To2017.Console/Options.cs

```diff
@@ -24,12 +24,16 @@ public class Options
 		[Option('o', "output-path", Default = false, HelpText = "Will not create a subfolder with the target framework in the output path")]
 		public bool NoTargetFrameworkToOutputPath { get; set; } = false;
 
+		[Option('z', "analyze-only", Default = false, HelpText = "Do only parsing and analysis steps, can be used even on already converted projects")]
+		public bool AnalyzeOnly { get; set; } = false;
+
 		public ConversionOptions ConversionOptions
 			=> new ConversionOptions
 			{
 				KeepAssemblyInfo = AssemblyInfo,
 				TargetFrameworks = TargetFrameworks?.ToList(),
 				AppendTargetFrameworkToOutputPath = !NoTargetFrameworkToOutputPath,
+				AnalyzeOnly = AnalyzeOnly,
```

Comment:
This has nothing to do with this PR but shouldn't this assign DryRun too?

Reply:
I don't actually know, but in Program only original Options fields are used so no problem for now. I agree this is quite strange.

Reply:
That's no problem actually since there is no DryRun in `ConversionOptions`. :)